tests/utils/test_databricks_utils.py

```diff
@@ -18,15 +20,18 @@
     check_databricks_secret_scope_access,
     get_databricks_host_creds,
     get_databricks_runtime_major_minor_version,
+    get_dbconnect_udf_sandbox_info,
     get_mlflow_credential_context_by_run_id,
     get_workspace_info_from_databricks_secrets,
     get_workspace_info_from_dbutils,
     get_workspace_url,
     is_databricks_default_tracking_uri,
     is_running_in_ipython_environment,
 )
+from mlflow.utils.os import is_windows
 
 from tests.helper_functions import mock_method_chain
+from tests.pyfunc.test_spark import spark  # noqa
```

Comment:
```suggestion
from tests.pyfunc.test_spark import spark  # noqa: <ID>
```

Please add the rule ID